rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -376,6 +379,9 @@ def Generate_LC_xml(fileName, config):
     """
     Generate an LC xml file
     """
+    import pdb
```

Comment:
@viduship do we need this
if its for debugging purpose while testing, please remove..

Reply:
yes, removed it.